scripts/apitypings/main.go

```diff
@@ -88,7 +90,8 @@ func TypeMappings(gen *guts.GoParser) error {
 	gen.IncludeCustomDeclaration(map[string]guts.TypeOverride{
 		"github.com/coder/coder/v2/codersdk.NullTime": config.OverrideNullable(config.OverrideLiteral(bindings.KeywordString)),
 		// opt.Bool can return 'null' if unset
-		"tailscale.com/types/opt.Bool": config.OverrideNullable(config.OverrideLiteral(bindings.KeywordBoolean)),
+		"tailscale.com/types/opt.Bool":           config.OverrideNullable(config.OverrideLiteral(bindings.KeywordBoolean)),
+		"github.com/hashicorp/hcl/v2.Expression": config.OverrideLiteral(bindings.KeywordUnknown),
```

Comment:
I was leaving a few comments about `guts`, and honestly the syntax for overriding types is just... unfortunate right now. 

I ran out of time when working on guts, and threw in type overrides in the current state just to make things possible, but it's not easy to use.

So I just made a PR that makes the changes I was proposing: https://github.com/coder/coder/pull/17339

One day I'll fixup guts and make this easier to use.

Reply:
That is fine